forge/db/views/Project.js

```diff
@@ -33,7 +33,14 @@ module.exports = {
 
         const settingsHostnameRow = proj.ProjectSettings?.find((projectSettingsRow) => projectSettingsRow.key === KEY_HOSTNAME)
         result.hostname = settingsHostnameRow?.value || ''
-
+        if (proj.Application) {
+            result.application = {
```

Comment:
Should this also use `app.db.views.Application.application`?

Reply:
`app.db.views.Application.application` is intended to be the full view of the application.

Here we only want a summary - as we do in the later lines that add a team summary without deferring to a specific view.

Reply:
This feels like something that should be centralised e.g. `app.db.views.Application.application(application, summary: true)` rather than spreading throughout the app

Reply:
I have pushed a change to introduce `applicationSummary` in line with the `teamSummary` view we already had. In doing so, I've hit the reason for why they weren't currently used. In *this* code path, `proj.Application` has already been converted to a vanilla object - it is no longer a database model object. The summary view was intended for use against the database model object.

I've updated the view to check if its a vanilla object or not and to do the required conversion.

It isn't perhaps the cleanest approach, but it works for now and shouldn't hold up progress on the broader Application work.